elements/urth-viz-table/urth-viz-table.html

```diff
@@ -235,7 +235,7 @@
                 // parse columns using data's first row
                 // in case user provides missing column headers
                 var datarows = this.datarows;
-                console.assert(datarows);
+                this._assert(datarows);
```

Comment:
BTW... why are we using assert? Its say `Non-standard` [here](https://developer.mozilla.org/en-US/docs/Web/API/console/assert)